lib/slather/project.rb

```diff
@@ -61,15 +61,29 @@ def self.open(xcodeproj)
       proj
     end
 
+    def failure_help_string
+      "\n\tAre you sure your project is generating coverage? Try `slather setup your/project.xcodeproj`.\n\tDid you specify your Xcode scheme? (--scheme or 'scheme' in .slather.yml)\n\tIf you're using a workspace, did you specify it? (--workspace or 'workspace' in .slather.yml)"
```

Comment:
I like this, but I'd rather not have the `setup` command in here, because it is only relevant for folks using older Xcodes.